boa/src/builtins/number/mod.rs

```diff
@@ -400,6 +390,13 @@ impl Number {
                 .throw_range_error("radix must be an integer at least 2 and no greater than 36");
         }
 
+        // 5. If radixNumber = 10, return ! ToString(x).
+        // This part should use exponential notations for long integer numbers commented tests
+        if radix == 10 {
+            // return Ok(to_value(format!("{}", Self::to_number(this).to_num())));
```

Comment:
Should we remove these comments?

Reply:
> Should we remove these comments?

Yes. the comments are deprecated, but maybe we should keep the `5. If radixNumber = 10, return ! ToString(x).` because it is taken from the spec.